src/app/dashboard/dashboard.component.html

```diff
@@ -4,17 +4,16 @@
     href="https://fonts.googleapis.com/icon?family=Material+Icons"
   />
 </head>
-
 <div class="flex-container">
   <div class="balance">
     <div>
       <app-donut-chart></app-donut-chart>
     </div>
     <div>
       <p>Godziny Przepracowane: {{ balanceData.hoursWorked }}</p>
-      <p>Minuty Przepracowane:{{ balanceData.minutesWorked }}</p>
-      <p>Godziny Do Przepracowania:{{ balanceData.hoursLeft }}</p>
-      <p>Minuty Do Przepracowania:{{ balanceData.minutesLeft }}</p>
+      <p>Minuty Przepracowane: {{ balanceData.minutesWorked }}</p>
+      <p>Godziny Do Przepracowania: {{ balanceData.hoursLeft }}</p>
+      <p>Minuty Do Przepracowania: {{ balanceData.minutesLeft }}</p>
```

Comment:
Tutaj fajnie byłoby zaprezentować dane w nieco bardziej skróconej formie:

```suggestion
      <p>Godziny Do Przepracowania: {{ balanceData.hoursLeft }}: {{ balanceData.minutesLeft }}</p>
```

Reply:
Te dane Józek już zmienił na swoim pr